app/models/balance.rb

```diff
@@ -0,0 +1,4 @@
+class Balance < ApplicationRecord
+    validates :value, presence: true, 
+                       numericality: { only_integer: true }
```

Comment:
Incorrect indentation level :)
```suggestion
                      numericality: { only_integer: true }
```